### ISSUE TITLE
Retrieve old chats over imap on first login

### ISSUE THREAD

**Issue by Jikstra** [deltachat/deltachat-core#493]
Maybe it's nice to be able to import old chats by downloading them from imap.
Related desktop issue: https://github.com/deltachat/deltachat-desktop/issues/494

**Comment by r10s** [comment]
yeah, this makes some sense, however, i would not call this a "bug" but a "feature proposal" :)
